src/Symfony/Component/Routing/Route.php

```diff
@@ -376,6 +376,10 @@ public function setDefaults(array $defaults)
      */
     public function addDefaults(array $defaults)
     {
+        if (isset($defaults['_locale']) && $this->isLocalized()) {
+            unset($defaults['_locale']);
```

Comment:
What about throwing? (same in other places)

Reply:
It would be a BC break, and more importantly it would forbid legit use cases where one imports several routes and some of them are localized.

Reply:
I don't understand how this is breaking BC while we are already making it a noop here. Instead of legit use I see a silent failure, which could be reported at compile time for something wrongly configured.

Reply:
There is no error on the userland side, reporting an exception would break legit use cases as I mentioned also. This is a bug, not a userland issue.